website/docs/r/chime_voice_connector_group.html.markdown

```diff
@@ -0,0 +1,71 @@
+---
+subcategory: "Chime"
+layout: "aws"
+page_title: "AWS: aws_chime_voice_connector_group"
+description: |-
+  Creates an Amazon Chime Voice Connector group under the administrator's AWS account.
+---
+
+# Resource: aws_chime_voice_connector_group
+
+Creates an Amazon Chime Voice Connector group under the administrator's AWS account. You can associate Amazon Chime Voice Connectors with the Amazon Chime Voice Connector group by including VoiceConnectorItems in the request.
```

Comment:
`Connectors` is capitalized here, but not in the log/error messages.

Reply:
I have updated the WARN and ERROR messages instead. Amazon documentation have it capitalized.